### DISCUSSION TITLE
How to enable 4-Bit Cache (Q4 Cache)?

### DISCUSSION THREAD

**Discussion by houmie** [PygmalionAI/aphrodite-engine#457] [Q&A] [answered]
Does Aphrodite-engine support 4-bit cache so that large models use less GPU memory without affecting the performance?

On RunPod's Aphrodite template I noticed these two settings:
<img width="885" alt="Screenshot 2024-05-08 at 08 44 04" src="https://github.com/PygmalionAI/aphrodite-engine/assets/1832933/54d56dfe-87aa-4a60-ac07-00bbf3c7516e">

I believe KVCACHE : fp8_e5m2 indicates the good old 8-Bit cache.
But I'm not sure what the --load-in-4bit means in this context.

Thanks

**Comment by tavviv** [comment, marked as answer]
Aphrodite does not support 4-bit cache.

--load-in-4bit means the model will be automatically quanted down to 4bit using smoothquant. It'll use the system RAM to do this.
